### DISCUSSION TITLE
article/spyxfamily

### DISCUSSION THREAD

**Discussion by giscus** [thedavidweng/NotionNext#7] [Announcements]
# article/spyxfamily

某天下午的饭点，我点开了间谍过家家并连续观看了三集，然后决定在剧情正式展开之前停止观看并写一篇锐评的稿子。

https://blog.davidweng.tk/article/spyxfamily

**Comment by Rain-yiyu** [comment]
感觉还是挺不错的 日常消遣吧

**Reply by thedavidweng** [reply]
又是一部被捧到不属于它的高度（佳作）的良作

**Comment by ukityh** [comment]
很赞同你说的23333主角塑造得都很纸片，男女主cp属于工业糖精（利益相关我喜欢bg所以去看的），一度觉得能霸权是不是因为大家喜欢间谍/杀手这种标签，或者阿尼娅被画得很可爱之类的，剧情真的很尬，有点网文青春校园频道的意思

**Reply by thedavidweng** [reply]
时势造英雄
